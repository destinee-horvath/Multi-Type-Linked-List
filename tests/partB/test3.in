NEW 5
a
b
c
d
e
DELETE 0 0
DELETE 0 0
DELETE 0 0 
DELETE 0 0
DELETE 0 0
DELETE 0 0
NEW 2
begin
end
INSERT 1 1 new insert
INSERT 1 1 before new insert
INSERT 0 0 4
INSERT 0 1 1000
INSERT 0 2 aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
INSERT 1 2 5
INSERT 1 3 *
TYPE 0
TYPE 1