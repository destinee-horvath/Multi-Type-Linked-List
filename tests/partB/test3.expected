List 0: a -> b -> c -> d -> e
List 0: b -> c -> d -> e
List 0: c -> d -> e
List 0: d -> e
List 0: e
List 0: 
INVALID COMMAND: DELETE
List 1: begin -> end
List 1: begin -> new insert -> end
List 1: begin -> before new insert -> new insert -> end
List 0: 4
List 0: 4 -> 1000
List 0: 4 -> 1000 -> aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
List 1: begin -> before new insert -> 5 -> new insert -> end
List 1: begin -> before new insert -> 5 -> * -> new insert -> end
int -> int -> string
string -> string -> int -> char -> string -> string
